loom/script/reflection/lsAssembly.h

```diff
@@ -67,12 +67,7 @@ class Assembly {
     // the assemblies we reference
     utArray<utString> references;
 
-    Type **ordinalTypes;
-
-    /*
-     * Closes the assembly, cleaning up associated hashes and Types
-     */
-    void close();
+    utArray<Type*> *ordinalTypes;
```

Comment:
This is a key performance sensitive path so I want to get some more details on this change. Why the change to utArray? Does it affect lookup speed/error checking?